sdlib/d/gc/thread.d

```diff
@@ -24,10 +24,13 @@ void createProcess() {
 	registerTlsSegments();
 }
 
-void createThread() {
+void createThread(bool isGCThread) {
```

Comment:
This could also be a template.

Reply:
Yeah, I think that it's a hard thing to name because it's so specific to this implementation. I can change to a template param.